consolida.py

```diff
@@ -138,13 +138,18 @@ def parse_state_file(self, response):
                     "deaths": deaths,
                 }
                 result.append(row)
+
+        result = sorted(result, key=lambda row:row["date"])
```

Comment:
Essa linha está incorreta. Para que o `groupby` funcione abaixo, a lista necessita estar ordenada pela `row_key` ([veja na documentação do Python](https://docs.python.org/3/library/itertools.html#itertools.groupby)). Também daria para fazer o sort inplace.
Sugiro trocar essa linha por:
```python
result.sort(key=row_key)
```
E dentro do `for key, row_list` ordenar por `date`.

Reply:
Opa beleza, esse arquivo não estava ordenando pela key=row_key antes, isso é normal?

Reply:
Não lembro por que não estava. Você adiciona?
Talvez fosse melhor a gente dividir esse método em várias funções menores e fazer testes automatizados para cada uma delas (no momento não consigo parar pra ver isso, mas se puder já ajuda também).

Reply:
Beleza.